train.py

```diff
@@ -0,0 +1,96 @@
+"""Script for training a tensorflow model on audio datasets.
+
+Usage:
+    python train.py --dataset=fma
+"""
+
+import argparse
+import logging
+
+import tensorflow as tf
+
+from datasets import fma
+
+MINI_BATCH_SIZE = 32
+LEARNING_RATE = .5
+
+
+def load_data(dataset, split="training", size="small"):
+    """Load data into trainable iterator."""
+    logging.info("Loading dataset %s" % dataset)
+    # TODO(amy): build support for cross validation rather than
+    # static train/test split
+    if dataset == 'fma':
+        path_to_data = 'raw_data'
+        dataset, all_labels = fma.read(path_to_data, split, size)
+        # Shuffle with shuffle buffer 10x the batch size
+        dataset = dataset.shuffle(MINI_BATCH_SIZE * 10)
+        batched_dataset = dataset.batch(MINI_BATCH_SIZE)
+        # Create iterator that will make one pass through the dataset
+        iterator = batched_dataset.make_one_shot_iterator()
+        # Returns placeholder tensors (features and labels) representing
```

Comment:
placeholder has a specific meaning in TF (tf.placeholder), so I might phrase this differently. What about:

Returns features and labels dictionaries containing tensors from this mini-batch. See `fma.read` for details on the tensors in these dictionaries.